nixie/unittests/test_get_renderer_cmd.py

```
@@ -0,0 +1,253 @@
+        monkeypatch.setattr(shutil, "which", _which_nothing)
+
+        assert find_merman_cli() is None
+
+
+class TestResolveRenderer:
+    """Resolution of the requested renderer choice to a backend."""
+
+    def test_auto_prefers_merman(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        fake_home_cwd: Path,
```

Ruff (0.15.15)

[warning] Unused method argument: `fake_home_cwd`

(ARG002)